setup.py

```diff
@@ -36,29 +36,26 @@ def load_about():
     license="AGPLv3",
     author="Emad Rad",
     author_email="codewithemad@gmail.com",
-    description="Tutor plugin for WooCommerce",
+    description="Tutor plugin for WordPress.",
     long_description=load_readme(),
     long_description_content_type="text/x-rst",
     packages=find_packages(exclude=["tests*"]),
     include_package_data=True,
-    python_requires=">=3.8",
-    install_requires=["tutor>=14.0.0,<19.0.0"],
+    python_requires=">=3.7",
+    install_requires=["tutor>=15.0.0,<19.0.0", "importlib_resources", "pylint"],
     extras_require={
         "dev": [
-            "tutor[dev]>=14.0.0,<19.0.0",
-        ]
-    },
-    entry_points={
-        "tutor.plugin.v1": [
-            "wordpress = tutorwordpress.plugin"
+            "tutor[dev]>=15.0.0,<19.0.0",
```

Comment:
Is it necessary to use tutor[dev] in a standard installation?

Reply:
This is a common pattern for tutor plugins. where we install tutor[dev], if you wnat to install tutor-contrib-wordpress[dev] for better developer experience. for example: https://github.com/overhangio/tutor-notes/blob/75e64f506597158fcd850ce76c5715acf7b0a163/setup.py#L37

Reply:
I reviewed it, and you are right; that part is not in the standard installation but in the dev mode ✨